### ISSUE TITLE
Possible DUO code for GSR policy / aggregate statistics

### ISSUE THREAD

**Issue by solideoglori** [EBISPOT/DUO#37]
Would we like to consider a DUO code to represent the following policy from the NIH?
https://grants.nih.gov/grants/guide/notice-files/NOT-OD-19-023.html

We have heard a interest for the ability to tag a dataset when this policy is applied to it, which we believe would be beneficial to repository managers and/or DACs, and researchers in the future. 

Does it need genericized for contexts beyond the US?

**Comment by athorogood** [comment]
Is the idea to create a tag describing the nature of the dataset (e.g., this is summary results)? This would not really be in DUO's scope as DUO can apply agnostically to any scientific "resource". 

Or is the idea to create a tag for genomic summary results stating the results are publicly available (for health and research purposes)? Then isn't this already covered by existing DUO labels?

There is also the exception for sensitive studies (GSR can stay controlled access). Again, the way to implement DUO here is not to tag the data as "sensitive", but rather to tag the GSR dataset as subject to controlled access. 

In short, I think the solution here is some guidance on how DUO would be implemented in such a scenario, rather than a new tag.

**Comment by lymanrodriguez** [comment]
I agree that this is likely a question about how to implement DUO in a way that accommodates the US policy.

**Comment by solideoglori** [comment]
Great point, thanks for the feedback. We will close this issue and take note to make implementation notes regarding this for future guidance/questions.
